scarb/tests/package.rs

```diff
@@ -1525,3 +1526,105 @@ fn package_with_publish_disabled() {
             [..]Packaged [..] files, [..] ([..] compressed)
         "#});
 }
+
+#[test]
+fn package_with_package_script() {
+    let t = TempDir::new().unwrap();
+
+    #[cfg(not(windows))]
+    let script_code = indoc! { r#"cargo build --release && mkdir -p ../scarb/cairo-plugin && cp target/release/libfoo.so ../scarb/cairo-plugin"#};
+
+    #[cfg(windows)]
+    let script_code = indoc! { r#"cargo build --release && mkdir -p ../scarb/cairo-plugin && cp target/release/libfoo.dll ../scarb/cairo-plugin"#};
```

Comment:
I think the users are not supposed to use relatives paths like `../scarb/cairo-plugin` or `../scarb/cairo-plugin`, and binaries should not be copied somewhere outside of the cwd tree.

The binaries imo should either:
1. Be required to be placed in `target/scarb/cairo-plugin/`
2. Be automatically located regardless of their path and then placed in the correct directory